flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/NXSystemMessages.js

```diff
@@ -72,5 +72,5 @@ const NXFictionalMessages = {
     noRefWpt:               new McduMessage("NO REF WAYPOINT", false, false),
     noWptInfos:             new McduMessage("NO WAYPOINT INFOS", false, false),
     emptyMessage:           new McduMessage(""),
-    weightUnitChanged:      new McduMessage("UNIT CHANGED RELOAD A/C", true, true)
+    reloadPlaneApply:       new McduMessage("RELOAD A/C TO APPLY", true, false)
```

Comment:
Why did you changed it to a type I message?
I'd assume this message has be actively processed by the pilot.

Reply:
Good catch. I missed that. 😞